### PR TITLE
Removed code block from Keploy developer section in keploy website la…

### PR DESCRIPTION
**Pull request by maskedeman**
…nding page #27

### CODE REVIEW

File: components/FeatureMoblieView.tsx [hunk 1]
```diff
@@ -227,15 +227,7 @@ export default function FeaturesMobileView() {
...
-                            <code>
-                              $ curl -O
-                              https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh
-                              && source keploy.sh
-                            </code>
-                          </pre>
-                        </div>
+                       
                         <div className="absolute right-3 bottom-2">
                           <button
```
Comment (nehagup) [inline components/FeatureMoblieView.tsx]: @maskedeman is this still required?

**Review by Hermione2408 — CHANGES_REQUESTED** [review]
Hey @maskedeman, we need to remove the entire code block, including the copy button, from all screen sizes as it's no longer required.

File: components/FeatureMoblieView.tsx (hunk 1 above)
Comment (maskedeman) [reply components/FeatureMoblieView.tsx]: > @maskedeman is this still required?

Sorry, my bad.

**Comment by maskedeman** [comment]
> Hey @maskedeman, we need to remove the entire code block, including the copy button, from all screen sizes as it's no longer required.

Ok, will do it.

File: components/FeatureMoblieView.tsx [hunk 2]
```diff
@@ -227,32 +213,8 @@ export default function FeaturesMobileView() {
                         ref={codeRef}
```
Comment (Hermione2408) [inline components/FeatureMoblieView.tsx]: Hey @maskedeman this is also not required

File: components/FeatureMoblieView.tsx [hunk 3]
```diff
@@ -19,7 +19,6 @@ export default function FeaturesMobileView() {
 
   const tabs = useRef<HTMLDivElement>(null);
   const codeRef = useRef<HTMLDivElement>(null);
```
Comment (Hermione2408) [inline components/FeatureMoblieView.tsx]: @maskedeman please check if this is still required?

**Review by Hermione2408 — CHANGES_REQUESTED** [review]

File: components/FeatureMoblieView.tsx (hunk 3 above)
Comment (maskedeman) [reply components/FeatureMoblieView.tsx]: Thank you for pointing out.

**Review by Hermione2408 — CHANGES_REQUESTED** [review]
@maskedeman please remove the unused imports too

**Review by Hermione2408 — APPROVED** [review]
LGTM! @nehagup

**Review by nehagup — APPROVED** [review]

**Pull request merged**